ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml

```diff
@@ -20,17 +16,9 @@ build_root:
 images:
 - dockerfile_path: Dockerfile.rhel7
   from: base
```

Comment:
This line overwrites the Dockerfile's `FROM` directive with the `base` image. Please remove it as well